bin/regression-wally

```diff
@@ -50,7 +50,7 @@ tests_buildrootboot = [
     ]
 
 tests_buildrootbootlockstep = [
-                    ["buildroot", ["buildroot"], [f"+INSTR_LIMIT=600000000 --lockstep"], # boot entire buildroot Linux to login prompt
+                    ["buildroot", ["buildroot"], {'args': [f"+INSTR_LIMIT=600000000 --lockstep"], 'params': ["STD_LOG=0"]}, # boot entire buildroot Linux to login prompt
```

Comment:
This isn't interpreted by regression-wally correctly. It attempts to output the following command based on this, which is not valid for wsim.
```
wsim --sim questa  buildroot buildroot --args args params
```

The correct command for what you're trying to attempt here would be:
```
wsim --sim questa buildroot buildroot --args "+INSTR_LIMIT=600000000" --lockstep --params "STD_LOG=0"
```

Reply:
that's weird, it works on my end

Reply:
If I checkout this PR I get the following:
```
$ regression-wally --buildroot
usage: wsim [-h] [--elf ELF] [--sim {questa,verilator,vcs}]
            [--tb {testbench,testbench_fp}] [--gui] [--ccov] [--fcov]
            [--args ARGS] [--params PARAMS] [--vcd] [--lockstep]
            [--locksteplog LOCKSTEPLOG] [--lockstepverbose] [--covlog COVLOG]
            [--rvvi]
            config [testsuite]
wsim: error: unrecognized arguments: params
grep: /home/jcarlin/cvw-new/sim/questa/logs/buildroot_uart.out: No such file or directory
grep: /home/jcarlin/cvw-new/sim/questa/logs/buildroot_uart.out: No such file or directory
grep: /home/jcarlin/cvw-new/sim/questa/logs/buildroot_uart.out: No such file or directory
wsim --sim questa  buildroot buildroot --args args params > /home/jcarlin/cvw-new/sim/questa/logs/buildroot_buildroot.log: Failures detected in output
  Check /home/jcarlin/cvw-new/sim/questa/logs/buildroot_uart.out
```